website/assets/sass/_searchview.scss

```diff
@@ -6,13 +6,56 @@
 @import "../../../node_modules/react-select/dist/react-select.css";
 
 
+$searchview-padding-x: 20px;
+$searchview-padding-y: 24px;
+
+$searchview-header-height: 50px;
+$searchview-heading-bottom-margin: $searchview-padding-y / 2;
+$searchview-body-text-bottom-margin: $searchview-heading-bottom-margin * 2;
+
 $search-inner-height: 550px;
 $results-view-max-height: 410px;
 $scroll-max-height: $results-view-max-height - 46;
 
+// Abstract base class for various specialized header toggle buttons.
+button.btn--header-toggle {
+    @include display(flex);
+    @include flex-direction(row);
+    @include align-items(center);
+    background-color: transparent;
+    color: white;
+    font-size: map-get($map-font-sizes, 'base');
+    font-family: $sans-font-family;
+    text-transform: uppercase;
+    padding: 0;
+
+    > span {
+        flex: 0 1 auto;
```

Comment:
Not sure if this is a nitpick or not, but the word FILTER in the span is centered vertically against the icon in the spec, but appears to be aligned on the bottom here. Adding 1px padding on the bottom made it look closer to me, not sure if that's the best way to do it though.